### ISSUE TITLE
Volume buttons do not control volume when unset for shutter on Android

### ISSUE THREAD

**Issue by polarbearing** [mapillary/mapillary_issues#1488]
While the usage of the volume buttons as shutter can be disabled as in #222, they do not control the volume.
(Android, long term issue also present in 1.52)

I have unset the shutter function, hoping to be able to control the volume, in particular to lower the volume to reduce the shutter sound (which the app cannot disable in my device). However even when unset, the app captures the volumeUp/Down event, but does not apply that to the volume setting:

Thu Nov 05 12:43:18 CET 2015 : app.mapillary.android.activity.CameraActivity  - Camera button pressedKeyEvent { action=ACTION_UP, keyCode=KEYCODE_VOLUME_DOWN, scanCode=114, metaState=0, flags=0x8, repeatCount=0, eventTime=1272227660, downTime=1272226131, deviceId=5, source=0x101 }
Thu Nov 05 12:43:19 CET 2015 : app.mapillary.android.activity.CameraActivity  - Camera button pressedKeyEvent { action=ACTION_DOWN, keyCode=KEYCODE_VOLUME_UP, scanCode=115, metaState=0, flags=0x8, repeatCount=0, eventTime=1272228226, downTime=1272228226, deviceId=5, source=0x101 }

**Comment by peterneubauer** [comment]
Since this feature is just useful for manual mode maybe we should disable this altogether to avoid confusion?

**Comment by peterneubauer** [comment]
So, in the latest builds the volume buttons will call the default behavior when the preference is deactivated (v 1.54+). Not sure what that behavior is on different phones.

**Comment by polarbearing** [comment]
>  I am not sure if lowering the system volume changes something about the camera shutter sound

Well it does for me, and it is the only chance to reduce or mute the ugly sound. Currently it works in the menus, but not yet in camera mode. 

I tested explicitely, pressing the volume keys 10 times in the menus, they do not at all show up in the log, but show the volume slider and adjust the volume, and each button press has a 'bip' noise that depends on the volume. 

I switched to camera mode and took a picture to see the step in the log.

Now in camera mode, the button presses do show up in the log as below, the make the 'bip' with constant level, do not show the volume slider and do not change the volume. 

```Mon Nov 23 14:01:28 CET 2015 : app.mapillary.android.activity.CameraActivity  - Camera button pressedKeyEvent { action=ACTION_DOWN, keyCode=KEYCODE_VOLUME_DOWN, scanCode=114, metaState=0, flags=0x8, repeatCount=0, eventTime=2104898691, downTime=2104898691, deviceId=5, source=0x101 }```
